Modules/Synchronization/src/Synchronization.Application/IntegrationEvents/Incoming/IdentityDeletionProcessStarted/IdentityDeletionProcessStartedIntegrationEventHandler.cs

```
@@ -22,7 +22,7 @@ public IdentityDeletionProcessStartedIntegrationEventHandler(ISynchronizationDbC
 
     public async Task Handle(IdentityDeletionProcessStartedIntegrationEvent integrationEvent)
     {
-        var payload = new { integrationEvent.DeletionProcessId };
+        var payload = new { DeletionProcessId = integrationEvent.DeletionProcessId };
```

You should be able to ignore this error with a `#pragma warning disable`.